scripts/create_newcase

```diff
@@ -168,11 +164,19 @@ OR
     if args.input_dir is not None:
         args.input_dir = os.path.abspath(args.input_dir)
 
+    if args.ninst_couplers:
+        ncouplers = args.ninst
+        ninst = 1
```

Comment:
Are you sure about that? That doesn't make sense to me. I think this misunderstanding (on one of our parts) is the cause for the different opinions on the best user interface here. I can't see any rationale for having multiple coupler instances with only one of each component instance. @mvertens or @gold2718 can you chime in here to say which is right?